index.js

```diff
@@ -2,8 +2,11 @@
 var objectAssign = require('object-assign');
```

Comment:
This is odd, `object-assign` is not used. Or maybe I'm looking over it. If it is used, use `Object.assign` and drop the dependency.